doc/sphinxman/source/optking.rst

```diff
@@ -262,45 +260,279 @@ For bends, the corresponding keyword is "frozen_bend".
+
+.. code-block:: none
+
+   memory 4GB 
+   molecule mol {
+       0 1 
+       O   -0.5026452583       -0.9681078610       -0.4772692868
+       H   -2.3292990446       -1.1611084524       -0.4772692868
+       H   -0.8887241813        0.8340933116       -0.4772692868
+       --  
+       0 1 
+       C    0.8853463281       -5.2235996493        0.5504918473
+       C    1.8139169342       -2.1992967152        3.8040686146
+       C    2.8624456357       -4.1143863257        0.5409035710
+       C   -0.6240195463       -4.8153482424        2.1904642137
+       C   -0.1646305764       -3.3031992532        3.8141619690
+       C    3.3271056135       -2.6064153737        2.1669340785
+       H    0.5244823836       -6.4459192939       -0.7478283184
+       H    4.0823309159       -4.4449979205       -0.7680411190
+       H   -2.2074914566       -5.7109913627        2.2110247636
+       H   -1.3768100495       -2.9846751653        5.1327625515
+       H    4.9209603634       -1.7288723155        2.1638694922
+       H    2.1923374156       -0.9964630692        5.1155773223
+       nocom
+       units au
+   }
+   
+   set {
+       basis 6-31+G 
+       frag_mode MULTI
+   }
+   
+   optimize("mp2")
+
+.. Warning:: The molecule input for psi4 has no effect upon optking, expect to provide Cartesian
+    coordinates. Specifying independent fragments with the `--` seperator, will not trigger 
+    optking to add specific interfragment coordinates. Use |optking__frag_mode| `multi`.
+
+* Specify the reference points to use for coordinates via |optking__frag_ref_atoms|.
+  Each list corresponds to a fragment. A list of indices denotes a linear combination
+  of the atoms. In this case, the first reference point for the second fragment is the center
+  of the benzene ring.
+
+.. code-block:: none
+
+   memory 4GB 
+   molecule mol {
+       0 1 
+       O   -0.5026452583       -0.9681078610       -0.4772692868
+       H   -2.3292990446       -1.1611084524       -0.4772692868
+       H   -0.8887241813        0.8340933116       -0.4772692868
+       --  
+       0 1 
+       C    0.8853463281       -5.2235996493        0.5504918473
+       C    1.8139169342       -2.1992967152        3.8040686146
+       C    2.8624456357       -4.1143863257        0.5409035710
+       C   -0.6240195463       -4.8153482424        2.1904642137
+       C   -0.1646305764       -3.3031992532        3.8141619690
+       C    3.3271056135       -2.6064153737        2.1669340785
+       H    0.5244823836       -6.4459192939       -0.7478283184
+       H    4.0823309159       -4.4449979205       -0.7680411190
+       H   -2.2074914566       -5.7109913627        2.2110247636
+       H   -1.3768100495       -2.9846751653        5.1327625515
+       H    4.9209603634       -1.7288723155        2.1638694922
+       H    2.1923374156       -0.9964630692        5.1155773223
+       nocom
+       units au
+   }
+   
+   set {
+       basis 6-31+G 
+       frag_mode MULTI
```

Comment:
```suggestion
       frag_mode MULTI
       The line below specifies the reference points that will be used to construct the interfragment coordinates between the two fragments (called A and B).  The format is the following:
       [[A-1], [A-2], [A-3]], [[B-1], [B-2], [B-3]]
 In terms of atoms within each fragment, the line below chooses, for water:

H3 of water for the first reference point, O1 of water for the second reference point, H2 of water for the third reference point

and for benzene:

the mean of the positions of all the C atoms, C2, one of the Carbon atoms, C6, another one of the carbon atoms.
```

Reply:
@AlexHeide , I am concerned that this example may now not be correct.  In order to support trimers, etc., I changed the atoms numbering to the total atom number - not the number within the fragment.  See e.g., test_dimers_ne2.py for the simplest example that also uses frag_ref_atoms.

Reply:
Yep. Tests got updated but my examples didn't. Good catch.